packages/ui-api/src/with/call.tsx

```diff
@@ -219,18 +217,18 @@ export default function withCall<P extends ApiProps> (
           return;
         }
 
-        const [apiMethod, params, isSubscription, isConst] = info;
+        const [apiMethod, params, isSubscription, area] = info;
         const updateCb = (value?: any): void =>
           this.triggerUpdate(this.props, value);
 
         await this.unsubscribe();
 
         try {
-          if (isSubscription) {
+          if (isSubscription || area === 'derive' || (area === 'query' && (!at && !atProp))) {
```

Comment:
Yeap, probably cleaner here now since we pass the area back. (Could actually remove `isSubscription` in this case as well, and just make the area for that `subscribe` - since we pass empty anyway, may as well combine the 2)

Reply:
we also have https://github.com/polkadot-js/apps/pull/1416/files#diff-e2040f6b00b230fed0e435eb6386b9a3R161

Reply:
Yes, I saw that, which is what lead me to the question. So what I'm suggesting,

- line 162 : return `[fn, params, 'subscribe']` (combine `true` and the `''`)
- line 192: return `[apiSection[method], newParams, method.startsWith('subscribe') ? 'subscribe' : area]`
- line 227 (here): if (['derive', 'subscribe']includes(area) || (area === 'query' && ...`